test/e2e/e2e_test.go

```diff
@@ -99,6 +103,12 @@ var _ = Describe("controller", Ordered, func() {
 			err = utils.LoadImageToKindClusterWithName(projectimage, kindClusterName)
 			ExpectWithOffset(1, err).NotTo(HaveOccurred())
 
+			By("deploy nginx onto the cluster")
+			cmd = exec.Command("kubectl", "apply", "-f",
+				"test/e2e/deploy-ingress-nginx.yaml")
```

Comment:
is this going to be executing at the right context? Guess we'll find out if the test passes